examples/CustomCollectionView/Sample/ImageCellNode.m

```diff
@@ -36,7 +34,9 @@ - (id)initWithImage:(UIImage *)image
 
 - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
 {
-  return [ASInsetLayoutSpec insetLayoutSpecWithInsets:UIEdgeInsetsZero child:_imageNode];
+  return [ASInsetLayoutSpec insetLayoutSpecWithInsets:UIEdgeInsetsZero
+                                                child:[ASRatioLayoutSpec ratioLayoutSpecWithRatio:self.image.size.height/self.image.size.width
+                                                                                            child:_imageNode]];
```

Comment:
Just to demonstrate best practice, might be worth having a CGSize imageSize = at the top